libs/libvtrutil/src/vtr_bimap.h

```diff
@@ -92,6 +109,37 @@ class bimap {
     ///@brief Return true if the specified value exists
     bool contains(const V value) const { return find(value) != inverse_end(); }
 
+  public: //Constructors/Deconstructor
+    ///@brief default constructor required by compiler
+    bimap() = default;
+
+    ///@brief default deconstructor required by compiler
+    ~bimap() = default;
+
+    ///@brief copy constructor
+    bimap(const bimap<K, V, Map, InvMap>& other)
```

Comment:
Could this copy-constructor be `= default` ?
(how about the move constructor ?)

Reply:
Both can hold the C++ default definition; as I mentioned, I defined them to provide consistency for all constructors. I will remove their definitions and utilize the default one.